airflow/www/views.py

```diff
@@ -2307,7 +2307,7 @@ def version(self):
         # Get the Git repo and git hash
         git_version = None
         try:
-            with open("airflow/git_version") as f:
+            with open(os.path.join(*[settings.AIRFLOW_HOME, 'airflow', 'git_version'])) as f:
```

Comment:
We will need to iterate on this as we learn more about how different companies install. This feature could be the forcing function.